freshservice/README.md

```diff
@@ -0,0 +1,143 @@
+# Freshservice Tickets using Webhooks
+
+## Overview
+
+This guide shows you how to use the Webhooks integration to open new tickets in Freshservice when a monitor alerts.
+
+
+## Setup
+
+### Installation
+
+To begin, open the Webhooks integration tile, go to the Configuration tab, then scroll to the bottom form to add a new webhook.
+
+
+#### Name
+
+Provide your webhook with a name. This name is used in your monitor message (see Usage) with @webhook-<NAME>. For example, if you name your webhook freshservice, you can open a ticket from your monitor by mentioning @webhook-freshservice in the monitor message.
+
+#### URL
+
+Freshservice has 2 different versions of their API. This guide uses V2, but it is possible to use V1 with slight modifications to your JSON payload.
+
+In the URL field enter the following endpoint:
+
+`https://<YOUR_DOMAIN>.freshservice.com/api/v2/tickets`
+
+#### Custom Payload
+
+First, check the **Use custom payload** checkbox to use a custom JSON payload. If you don’t do this, the standard Datadog payload is sent to Freshservice resulting in a failed request.

```

Comment:
I don't think this button exist, you can just set the payload in the UI text area